examples/wordcloud_cn.py

```diff
@@ -0,0 +1,95 @@
+# - * - coding: utf - 8 -*-
+"""
+create wordcloud with chinese
+=======================
+
+Wordcloud is a very good tools, but if you want to create
+Chinese wordcloud only wordcloud is not enough. The file
+shows how to use wordcloud with Chinese. First, you need a
+Chinese word segmentation library jieba, jieba is now the
+most elegant the most popular Chinese word segmentation tool in python.
+You can use 'PIP install jieba'. To install it. As you can see,
+at the same time using wordcloud with jieba very convenient
+"""
+
+import jieba
+jieba.enable_parallel(4)
+# Setting up parallel processes :4 ,but unable to run on Windows
+from os import path
+from scipy.misc import imread
+import matplotlib.pyplot as plt
+# jieba.load_userdict("txt\userdict.txt")
```

Comment:
if this is not needed, please remove it.

Reply:
This command allows the user to import a custom word library using the txt file. If you think it's useless, I'll delete it.